docs/admin/config.rst

```diff
@@ -1463,10 +1463,10 @@ REGISTRATION_ALLOW_BACKENDS
 
 List of authentication backends to allow registration from. This only limits
```

Comment:
```suggestion
List of authentication back-ends to allow registration from. This only limits
```
Seems this has changed now https://en.wikipedia.org/wiki/Frontend_and_backend
which could mean changing all "back-end" to "backend" is warranted.
Did not consult the Weblate glossary.

Reply:
It is called "backend" in https://python-social-auth.readthedocs.io/en/latest/backends/index.html and https://docs.djangoproject.com/en/4.2/topics/auth/customizing/#authentication-backends, so lets stick with the same naming here as well.